### PR TITLE
Add random file create to test file require in libraries

### PR DESCRIPTION
**Pull request by KwisatzHaderach**

### CODE REVIEW

File: file/main.fmf [hunk 1]
```diff
@@ -2,4 +2,7 @@ summary: Example of a beakerlib library
 contact: Petr Šplíchal <psplicha@redhat.com>
 test: ./test.sh
 framework: beakerlib
-require: coreutils
+require:
+  - coreutils
+  - pattern: random_file.sh
```
Comment (psss) [inline file/main.fmf:7]: Just to clarify: In the spec we have:

> When referencing local files or directories use `type: file` and define list of paths relative to the fmf root directory.

So we don't expect `/scripts/random_file.sh` here but we plan to support matching the filename anywhere in the path, right?

Comment (KwisatzHaderach) [reply file/main.fmf:7]: Well it's a regex so anything goes, full relative path is not necessary. Maybe the description is a bit off.

Comment (psss) [reply file/main.fmf:7]: Ok, just wanted to clarify the intention.

**Review by psss — APPROVED** [review]
Looks good.

**Pull request merged**